app/renderer/components/preferences/payment/bitcoinDashboard.js

```diff
@@ -344,53 +244,13 @@ class BitcoinDashboard extends ImmutableComponent {
     return null
   }
 
-  bitcoinOverlayContent () {
-    return <iframe className={css(styles.modalOverlay__coinbaseOverlay__bodyWrapper__body__iframe)}
-      src={this.props.ledgerData.get('buyURL')}
-    />
-  }
-
   copyToClipboard (text) {
     aboutActions.setClipboard(text)
   }
 
-  onMessage (e) {
-    if (!e.data || e.origin !== config.coinbaseOrigin) {
-      return
-    }
-    if (e.data.event === 'modal_closed') {
-      if (this.buyCompleted) {
-        this.props.hideParentOverlay()
-        this.buyCompleted = false
-      } else {
-        this.props.hideOverlay()
-      }
-    } else if (e.data.event === 'buy_completed') {
-      this.buyCompleted = true
-    }
-  }
-
   render () {
-    window.addEventListener('message', this.onMessage.bind(this), false)
-    const ledgerData = this.props.ledgerData
-
     return <section data-test-id='bitcoinDashboard'>
       {
-        this.props.bitcoinOverlayVisible
-          ? <ModalOverlay
-            title={'bitcoinBuy'}
-            content={this.bitcoinOverlayContent()}
-            customDialogClasses={css(styles.modalOverlay__coinbaseOverlay)}
```

Comment:
Just in case please remove or edit https://github.com/NejcZdovc/browser-laptop/blob/73f9795019a37e5b94cd41eb42cc12857353ccc4/app/renderer/components/preferences/payment/bitcoinDashboard.js#L463-L501 if `modalOverlay__coinbaseOverlay` is no longer used.